foundations/javascript_basics/DOM_manipulation_and_events.md

```diff
@@ -2,7 +2,7 @@
 
 One of the most unique and useful abilities of JavaScript is its ability to manipulate the DOM. But what _is_ the DOM, and how do we go about changing it? Let's jump right in...
 
-### Learning Outcomes
+### Lesson Overview
 By the end of this lesson, you should be able to do the following:
```

Comment:
This line should also be updated in all lessons in the PR.

Similarly, the Knowledge Check sections should also be updated.

Reply:
Hi I have done the required changes, can u please take a look.